daml-lf/spec/daml-lf-1.rst

```diff
@@ -1609,6 +1697,15 @@ Specifically, a template definition is *coherent* if:
 * Its argument data type is not an argument to any other template.
 
 
+Exception coherence
```

Comment:
`Well-formed packages` section needs to be updated to include `Exception coherence`

Furthermore, I would move the extra condition as a separate bullet in the `Well-formed packages` as it is not directly related the Exception coherence

Reply:
I wasn't aware of this. Thanks for pointing me to it.